bin/linux-checksums.sh

```diff
@@ -1,15 +1,34 @@
 #!/usr/bin/env bash
 
+#
+# Wire
+# Copyright (C) 2018 Wire Swiss GmbH
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see http://www.gnu.org/licenses/.
+#
+
 set -eu -o pipefail
 
 SCRIPT_NAME="${0##*/}"
-SCRIPT_DIR="${0%/*}"
 
 GPG_TEMP_DIR=".gpg-temporary"
-GPG_TEMP_DIR2=".gpg-temporary/private-keys-v1.d"
+GPG_TEMP_KEYS_DIR="${GPG_TEMP_DIR}/private-keys-v1.d"
 PGP_SIGN_ID="D599C1AA126762B1"
 PGP_KEYFILE="${PGP_PRIVATE_KEY_FILE:-${PGP_SIGN_ID}.asc}"
 PGP_PASSPHRASE="${PGP_PASSPHRASE:-""}"
+SHRED_STATUS="available"
+BUILD_VERSION="${1}"
```

Comment:
Maybe it is good to move this on top to see that this is a needed parameter for the script